Dockerfile.arm32v7

```diff
@@ -0,0 +1,38 @@
+FROM alpine:latest AS builder
+
+RUN apk --update --no-cache add curl tar && \
+    curl -L https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-arm.tar.gz | tar zxvf - -C . && mv qemu-3.0.0+resin-arm/qemu-arm-static .
+
+FROM arm32v7/alpine:latest
+
+# Copy across the qemu binary that was downloaded in the previous build step
+COPY --from=builder /qemu-arm-static /usr/bin
+
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL org.label-schema.build-date=$BUILD_DATE \
```

Comment:
`org.label-schema` is the old version, please see the current `Dockerfile` for the new label schema.